### PR TITLE
[SMALLFIX] Enable Findbugs checking for uses of potentially uninitialized fields

### PR DESCRIPTION
**Pull request by aaudiber**
https://tachyon.atlassian.net/browse/TACHYON-1197

### CODE REVIEW

File: servers/src/main/java/tachyon/master/file/meta/InodeTree.java [hunk 1]
```diff
@@ -429,6 +429,9 @@ public void setPinned(Inode inode, boolean pinned) {
    * @return true if the given file id is the root id
    */
   public boolean isRootId(long fileId) {
+    if (mRoot == null) {
+      throw new IllegalStateException("Cannot call isRootId() before initializeRoot()");
+    }
```
Comment (calvinjia) [inline servers/src/main/java/tachyon/master/file/meta/InodeTree.java]: Should this just be a precondition?

**Comment by yupeng9** [comment]
Same comment as Calvin's, otherwise LGTM

File: servers/src/main/java/tachyon/master/file/meta/InodeTree.java (hunk 1 above)
Comment (aaudiber) [inline servers/src/main/java/tachyon/master/file/meta/InodeTree.java]: Done

File: servers/src/main/java/tachyon/master/file/meta/InodeTree.java [hunk 2]
```diff
@@ -429,6 +429,7 @@ public void setPinned(Inode inode, boolean pinned) {
    * @return true if the given file id is the root id
    */
   public boolean isRootId(long fileId) {
+    Preconditions.checkNotNull(mRoot, "Cannot call isRootId() before initializeRoot()");
```
Comment (gpang) [inline servers/src/main/java/tachyon/master/file/meta/InodeTree.java]: Should all precondition messages go in PreconditionMessage.java? What is the convention?

**Comment by gpang** [comment]
I just had a minor question. Thanks!

File: servers/src/main/java/tachyon/master/file/meta/InodeTree.java (hunk 2 above)
Comment (aaudiber) [inline servers/src/main/java/tachyon/master/file/meta/InodeTree.java]: Done

**Comment by gpang** [comment]
LGTM

**Pull request merged**